gradio/interface.py

```diff
@@ -219,12 +223,33 @@ def __init__(
         self.simple_server = None
         self.allow_screenshot = allow_screenshot
         
-        # For allow_flagging and analytics_enabled: (1) first check for 
-        # parameter, (2) check for environment variable, (3) default to True
+        # For analytics_enabled and allow_flagging: (1) first check for 
```

Comment:
unnecessary comment

Reply:
Disagree here. I know we're trying to reduce comments, but here I think the comment helps clear up the logic significantly